pkg/pipeline/transform/transform_network.go

```diff
@@ -119,7 +121,10 @@ func (n *Network) Transform(inputEntry config.GenericMap) (config.GenericMap, bo
 			}
 			kubernetes.EnrichLayer(outputEntry, rule.KubernetesInfra)
 		case api.OpReinterpretDirection:
-			reinterpretDirection(outputEntry, &n.DirectionInfo)
+			isInner, isSrcReporter := reinterpretDirection(outputEntry, &n.DirectionInfo)
+			if n.cache != nil {
+				return n.cache.DedupFlows(isInner, isSrcReporter, outputEntry)
```

Comment:
I was weirdly seeing no flow under layer=app filter and now I understand : this `return` breaks the rules loop processing :-)
It should be `continue`

Reply:
This case push flow to cache so you don't need to manage extra transform rules on it right ?
The goal is actually to break the pipeline here.

When the second flow will hit this stage, the first flow is retreived from cache and other transform rules are applied such as `layer`